src/main/java/jenkins/plugins/logstash/LogstashNotifier.java

```diff
@@ -35,14 +35,12 @@
 import hudson.tasks.Publisher;
 
 import java.io.IOException;
-import java.util.Collections;
-import java.util.Date;
+import java.io.OutputStream;
+import java.io.PrintStream;
+import java.util.Arrays;
 import java.util.List;
 
-import jenkins.model.Jenkins;
 import jenkins.plugins.logstash.persistence.BuildData;
-import jenkins.plugins.logstash.persistence.LogstashIndexerDao;
-import net.sf.json.JSONObject;
```

Comment:
You can also delete the following imports:

java.io.IOException
java.util.Arrays
java.util.List
jenkins.plugins.logstash.persistence.BuildData
org.apache.commons.lang.exception.ExceptionUtils